### PR TITLE
Internal: Docs - Experiments Manager [ED-4464]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: docs/core/experiments/manager.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Experiments Manager
+
```
Comment (nisan250) [inline docs/core/experiments/manager.md:2]: Maybe change the PR name from "Docs: Experiments Manager [ED-4464]"  to "New: Docs experiments Manager [ED-4464]"

File: docs/core/experiments/manager.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Technical Description:
+
+This is the Experiment Manager of Elementor. It registers and manages all the experimental features in Elementor.
+
+Each module (`Elementor\Core\Base\Module`) can register its own experiments using the static `get_experimental_data()` method.
+
+In addition, you can "talk" directly to the manager in order to register an experiment using `Plugin::instance()->experiments->add_feature()`.
+
+Using both of the above methods, you can set the experiment name, description, release status (alpha, beta, dev, etc.),
+default state (active / inactive) and more! You can even pass a callback to run when the feature state has changed!
```
Comment (nisan250) [inline docs/core/experiments/manager.md:27]: (active/inactive), and

**Review by nisan250 — CHANGES_REQUESTED** [review]

File: docs/core/experiments/manager.md (hunk 1 above)
Comment (StyleShit) [reply docs/core/experiments/manager.md:2]: Done.

File: docs/core/experiments/manager.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Experiments Manager
+
+Since some of our features are still being worked on, and to avoid any potential issues,
+we’ve given you the option of switching them on and off and testing how they impact your site.
+You’ll be able to try out these features in a safe environment and influence future releases.
```
Comment (matipojo) [inline docs/core/experiments/manager.md:5]:
```suggestion
we've given you the option of switching them on and off and testing how they impact your site.
You'll be able to try out these features in a safe environment and influence future releases.
```

File: docs/core/experiments/manager.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- [Introducing experiments in Elementor 3.1](https://elementor.com/blog/introducing-elementor-3-1/)
+
+
+## Technical Description:
+
+This is the Experiment Manager of Elementor. It registers and manages all the experimental features in Elementor.
+
+Each module (`Elementor\Core\Base\Module`) can register its own experiments using the static `get_experimental_data()` method.
+
+In addition, you can "talk" directly to the manager in order to register an experiment using `Plugin::instance()->experiments->add_feature()`.
```
Comment (matipojo) [inline docs/core/experiments/manager.md:24]: Add an example

**Review by matipojo — CHANGES_REQUESTED** [review]

File: docs/core/experiments/manager.md (hunk 3 above)
Comment (matipojo) [reply docs/core/experiments/manager.md:5]: Remove al `you` `we` etc.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**